Wikipedia/Code/WMFRandomSectionController.m

```diff
@@ -112,6 +112,9 @@ - (void)getNewRandomArticle {
     if (self.fetcher.isFetching) {
         return;
     }
+
+    [self.delegate controllerWillGetItems:self];
```

Comment:
this is because didSetItems causes the configureCell to fire. 

If the animations don't look good let me know - we can work it out.